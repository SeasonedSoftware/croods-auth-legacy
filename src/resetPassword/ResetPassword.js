import React from 'react'
import New from '../New'

export default props => {
  const { relativePath = '/auth/password', location } = props

  return (
    <New
      {...props}
      method="put"
      name="auth.resetPassword"
      path={relativePath}
      otherProps={{ location }}
    />
  )
}
