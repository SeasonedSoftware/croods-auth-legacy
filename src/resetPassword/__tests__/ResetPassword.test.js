import React from 'react'
import renderer from 'react-test-renderer'

import ResetPassword from '../ResetPassword'

jest.mock('../../New', () => props => <div {...props}>New</div>)

it('renders correctly', () => {
  const props = { foo: 'bar' }
  const tree = renderer.create(<ResetPassword {...props} />).toJSON()

  expect(tree).toMatchSnapshot()
})
