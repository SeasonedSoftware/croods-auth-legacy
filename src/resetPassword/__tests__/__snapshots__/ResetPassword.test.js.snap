// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  foo="bar"
  method="put"
  name="auth.resetPassword"
  path="/auth/password"
>
  New
</div>
`;
